src/components/Theme/tokens.scss

```
@@ -21,46 +40,74 @@
   --font-weight-subtitle: 600;
   --font-weight-body: 500;
 
-  /* Color Variables */
+  /* General UI element background color settings */
+  --osdk-color-border-surface-modal: rgba(var(--ods-color-border-input));
   --color-background: rgba(var(--ods-color-neutral-050));
   --color-warning: rgba(var(--ods-color-primary-500));
-  --color-error: rgba(220, 42, 42);
+  --color-error: rgba(var(--ods-color-background-negative));
```

No action needed, just curious why don't you use Castor Sass helper? So it's:

```suggestion
  --color-error: castor.color('background-negative');
```

---

No specific reason actually. I didn't know of the helper until it came up in the recent discussion. & then forgot to make the changes. 😅